pytential/linalg/proxy.py

```diff
@@ -504,9 +504,9 @@ def knl():
                     shape="nnbrindices"),
                 lp.GlobalArg("nodes", None,
                     shape=(lpot_source.ambient_dim, "nproxies + nnbrindices")),
-                lp.ValueArg("nsources", np.int),
-                lp.ValueArg("nproxies", np.int),
-                lp.ValueArg("nnbrindices", np.int),
+                lp.ValueArg("nsources", np.int64),
+                lp.ValueArg("nproxies", np.int32),
+                lp.ValueArg("nnbrindices", np.int32),
```

Comment:
How come these are `int32`?

Reply:
I set them to `int32` because I don't expect the number of proxy points to ever be that big, but I'm not opposed to setting it to `int64`.

Reply:
Ah, no, wait! This is the total number of proxy points, not just for one block, and that can get quite large.

I'll update it to `int64`.